modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase

```diff
@@ -3222,7 +3222,7 @@ definition {
 
 			Click(locator1 = "WorkflowAllItems#WORKFLOW_INSTANCE_ID");
 
-			while ((IsElementNotPresent(locator1 = "WorkflowAllItems#DETAIL_PAGE_SLA_PAUSED")) && (maxIterations = "25")) {
+			while ((IsElementNotPresent(locator1 = "WorkflowAllItems#DETAIL_PAGE_SLA_PAUSED")) && (maxIterations = "65")) {
```

Comment:
I changed the maxIterations to 65 so it guarantees that it does take about a minute for the status to change.

Reply:
Actually now that I think of it, why shouldn't we just do a Pause here? @kristinonias

Reply:
I've added a 60s pause and reduced the iteration to 20. It doesn't pass with just the pause alone.